ckb-bin/src/subcommand/daemon.rs

```diff
@@ -0,0 +1,63 @@
+use ckb_app_config::{DaemonArgs, ExitCode};
+use colored::*;
+use nix::sys::signal::{kill, Signal};
+use nix::unistd::Pid;
+use std::fs;
+use std::path::PathBuf;
+
+pub fn daemon(args: DaemonArgs) -> Result<(), ExitCode> {
+    let name = "ckb";
+    let pid_file = &args.pid_file;
+    if args.check {
+        // find the pid file and check if the process is running
+        match check_process(pid_file) {
+            Ok(pid) => {
+                eprintln!("{:>10} : {:<12} pid - {}", name, "running".green(), pid);
+            }
+            _ => {
+                eprintln!("{:>10} : {:<12}", name, "not running".red());
+            }
+        }
+    } else if args.stop {
+        kill_process(pid_file, "ckb")?;
+        fs::remove_file(pid_file).map_err(|_| ExitCode::Failure)?;
+    } else {
+        unimplemented!()
```

Comment:
should print help message instead of panic

Reply:
invalid option will be terminated with help message by the caller of this function.
I think it also OK to remove this assertion.